omero/sysadmins/whatsnew.txt

```diff
@@ -7,7 +7,17 @@ What's new for OMERO 5.3 for sysadmins
   the Ice Python bindings if your package manager does not provide the Ice
   Python packages.
 
-- All OMERO web apps can now be installed using ``pip install``.
+- :doc:`version-requirements` has been updated to reflect other changes in
+  version support for 5.3.0 and tentative plans for 5.4.0.
+  
+- All official OMERO web apps can now be installed using ``pip install``. If
+  you have been running Django 1.6.x prior to upgrading, you should remove and
+  reinstall your plugins when you upgrade.
```

Comment:
I  don't quite get the logical connection here between running now-unsupported Django (which may have been installed in all manner of ways) and reinstalling plugins but if it makes sense to web/python people then that's good with me! Might some of the previous plugins have irreversibly self-configured on the basis of the Django version they found?